packages/react-mui-store-file-viewer/src/components/StoreFileViewer/index.tsx

```diff
@@ -51,7 +53,17 @@ export const StoreDownloadFileViewer: FC<StoreFileViewerProps> = ({
       defaultSortBy={defaultSortBy}
       defaultShareLinkTitle={defaultShareLinkTitle}
     >
-      <StoreDownloadDataGrid {...props} />
+      <>
+        {configurations?.controls?.editorMode && (
```

Comment:
I wonder if maybe then a separate mode prop, it should just use if apiRef?.upload or apiRef?.folderCreate are provided, and only show the folder button if folderCreate is available

Reply:
Good idea, updated to use apiRef.folderCreate